package.json

```diff
@@ -58,7 +58,7 @@
     "simple-git-hooks": "^2.9.0",
     "sync-request": "^6.1.0",
     "tsx": "^3.13.0",
-    "typescript": "5.1.6",
+    "typescript": "^5.1.6",
```

Comment:
Let's not have `^` here because it will install `5.2.2` on install without `package-lock.json`.

Reply:
Who put that there...

Reply:
Fixed